cg/services/orders/store_order_services/store_pacbio_order_service.py

```diff
@@ -65,6 +65,12 @@ def _store_samples_in_statusdb(
         customer: Customer = self.status_db.get_customer_by_internal_id(
             customer_internal_id=customer_id
         )
+        status_db_order = Order(
+            customer=customer,
+            order_date=datetime.now(),
+            ticket_id=int(ticket_id),
+            workflow=Workflow(samples[0]["data_analysis"]),
```

Comment:
I just add this as in the microbial fastq is this way, but disregard if irrelevant
```suggestion
            workflow=Workflow.RAW_DATA,
```

Reply:
That is probably a better way. We really should rework the whole flow soon though